src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -313,6 +313,28 @@ ParamArray& MainWindow::get_settings()
     return m_settings;
 }
 
+QDockWidget* MainWindow::create_dock(const char* dock_name)
+{
+    QDockWidget* dock_widget = new QDockWidget(this);
+
+    QString object_name = QString(dock_name).toLower().split(' ').join("_");
+    dock_widget->setObjectName(object_name);
+    dock_widget->setWindowTitle(dock_name);
+    dock_widget->setAllowedAreas(Qt::BottomDockWidgetArea);
```

Comment:
yes to simulate behavior of python console and log. It can be changed in script. I can delete it if you think it shouldn't be default.